code/style.css

```diff
@@ -1,4 +1,188 @@
 html {
-  background: #0025ff;
-  color: #fff;
+    background: rgb(134, 128, 128)
+    color:black;
 }
+
+body {
+  margin:0px;
+  border-bottom: solid 5px rgb(46, 7, 7);
+}
+
+/***HEADER SECTION***/
+
+  .header1 {
+    display:flex;
+    justify-content:center;
+    align-items: center;
+    background-color:white;
+    padding: 0 10px; 
+    size: 50%;
+    border-bottom: solid 5px rgb(46, 7, 7);
+  }
+
+  .logo-pic {
+    border: 1px solid;
+    width:50px;
+    margin-left: 10px;
+    margin-right: 4px;
+  }
+
+  .qtimes {
+    margin-left: 2px;
+    margin-right: 5px;
+  }
+
+  .hamburger {
+    display: flex;
+    width: 50px;
+    height: 30px;
+    flex-direction: column;
+    justify-content: space-between;
+  }  
+  
+  .hamburger div {
+    width: 30px;
+    height: 5px;
+    background: black;
+  }
+
+nav {
+display:none;
+font-size: 20px;
+padding-left: 50px
+}
+
+nav a {
+  color: black;
+  text-decoration:none;
+}
+
+/***BIG NEWS SECTION***/
+
+h6 {
+  color: rgb(210, 26, 26);
+}
+
+.big-news-section {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  margin:15px 10px 30px 10px;
+  border-bottom: solid rgb(46, 7, 7);
+}
+
+.big-news-section img {
+  width: 100%;
+  box-shadow: 5px 10px rgb(46, 7, 7);
+}
+
+.big-news-section div {
+  padding-bottom: 20px;;
+}
+
+/*** OTHER NEWS SECTION ***/
+
+.other-news-section {
+display: flex;
+align-items: center;
+flex-direction: column;
+}
+
+.other-news {
+  width:95%;
+  margin: 0px 5px 30px 5px;
+  border-bottom: solid rgb(46, 7, 7);
+}
+
+.other-news img {
+  width: 100%;
+  box-shadow: 5px 10px rgb(46, 7, 7);
+}
+
+/*** MEDIA QERIES ***/
+
+@media (min-width: 668px) {
+  
+  /***HEADER SECTION***/
+
+  .header1 {
+    justify-content:flex-start;
+  }
+
+    .hamburger {
+      display: none;
+  }
+  
+  nav {
+    display: inline-block;
+  }
+
+  /***BIG NEWS SECTION***/
+
+  .big-news-section img {
+    margin: 20px 40px 5px 40px;
+    max-width: 80%;
+  }
+
+  .big-news-section div {
+    text-align: center;
+    padding: 0px 10px 15px 10px;
+    margin-bottom: 20px;
+    border-bottom: none;
+  }
+
+  /*** OTHER NEWS SECTION ***/
+
+  .other-news-section {
+  flex-wrap: wrap;  
+  flex-direction: row;
+  justify-content: space-around;
+  }
+
+.other-news {
+  width:40%;
+  margin: 0px 10px;
+  border-bottom:none;
+  }
+
+.news-text {
+  text-align: center;
+  padding-bottom: 20px;
+  } 
+}
+
+  @media (min-width: 950px) {
+
+   /***BIG NEWS SECTION***/
+
+.big-news-section {
+  flex-direction: row;
+  align-items: center;
+  margin: 20px 15px 30px 15px;
+  padding-bottom: 40px;
+  }
+
+.big-news-section img {
+  width: 40%;
+  }
+
+.big-news-section div{
+  padding:0px 30px 0px 0px;
+  border-bottom:none;
```

Comment:
this is already set in the media query above, line 131. No need to add it again as the rule above applies here as well

Reply:
I did remove some of the doubles, although I left the padding part as it was different